src/mam4xx/mo_sethet.hpp

```diff
@@ -0,0 +1,181 @@
+#ifndef MAM4XX_MO_SETHET_HPP
+#define MAM4XX_MO_SETHET_HPP
+
+#include <haero/math.hpp>
+#include <mam4xx/gas_chem.hpp>
+#include <mam4xx/mam4_types.hpp>
+#include <mam4xx/mo_chm_diags.hpp>
+#include <mam4xx/utils.hpp>
+
+namespace mam4 {
+
+namespace mo_sethet {
+
+// const int ktop = ConvProc::Config::ktop;
+// int ktop = 47; // BAD_CONSTANT: only true for nlev == 72
+constexpr Real avo = haero::Constants::avogadro;
+const Real pi = haero::Constants::pi;
+constexpr Real rgrav =
+    mo_chm_diags::rgrav; // reciprocal of acceleration of gravity ~ m/s^2
+constexpr const int gas_pcnst = gas_chemistry::gas_pcnst;
+const Real boltz_cgs = haero::Constants::boltzmann * 1.e7; // erg/K
+// number of vertical levels
+constexpr const int pver = mam4::nlev;
+
+// FIXME: BAD CONSTANT
+constexpr const Real mass_h2o = 18.0;   // mass of water vapor [amu]
```

Comment:
Is this molecular weight of H2O? We could use haero::Constant.

Reply:
looks like it could be the molecular weight of water in g/mol instead of kg/mol?

Reply:
it is g/mol. Thus, we will need to multiple by 1000 the values from  haero::Constant::molec_weight_h2o.  I also noted the values in haero is 
```
/// Molecular weight of water [kg/mol]
static constexpr Real molec_weight_h2o = 0.018016;
```
, which has a few extra digits.